### PR TITLE
Test collectstatic and compress as part of Travis build

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: templates/channels/channel_claim_verboice.haml [hunk 1]
```diff
@@ -1,4 +1,4 @@
--extends "channel/channel_claim_form.html"
+-extends "channels/channel_claim_form.html"
```
Comment (rowanseymour) [inline templates/channels/channel_claim_verboice.haml]: Found its first template problem already

**Comment by ericnewcomer** [comment]
How bout we move these checks (flake, migration, compress) to run before running the tests? I'd like the build to fail quickly if any of these fail.

**Comment by rowanseymour** [comment]
@ericnewcomer have reordered. Are we sure that a step failing does actually fail the entire test ?

**Comment by ericnewcomer** [comment]
Nope, don't think it does currently. But surely that should be possible.

**Comment by rowanseymour** [comment]
Maybe not https://github.com/travis-ci/travis-ci/issues/1066

**Pull request merged**